default.mk

```diff
@@ -22,19 +23,19 @@ docker-build:
 clean: check-docker
 	docker rmi $(IMAGE)
 
-KD = kubectl --namespace=deis
-upgrade:
-	${KD} patch ${RESOURCE_TYPE} deis-monitor-${SHORT_NAME} \
-		--type='json' \
-		-p='[ \
-			{"op": "replace", "path": "/spec/strategy", "value":{"type":"Recreate"}}, \
-			{"op": "replace", "path": "/spec/template/spec/containers/0/image", "value":"$(IMAGE)"}, \
-			{"op": "replace", "path": "/spec/template/spec/containers/0/imagePullPolicy", "value":"Always"} \
-		]'
-	@if [ "${RESOURCE_TYPE}" == "daemonset" ]; then \
-		echo "Deleting daemonset pods as they aren't yet recreated when daemonset is patched..."; \
-		${KD} delete $$(${KD} get pod -o name | grep "${SHORT_NAME}"); \
-	fi
+# KD = kubectl --namespace=deis
```

Comment:
should be good to delete these commented-out lines now, right?

if so, we can remove the `RESOURCE_TYPE` declarations in each sub-component `Makefile` as well...